### DISCUSSION TITLE
퍼블리시 에러

### DISCUSSION THREAD

**Discussion by hipsc159** [naverz/zepeto-studio-kor#1011] [5. Q&A]
안녕하세요.

퍼블리시 중에 에러가 있어 문의 드립니다.

이번이 처음 퍼블리시 하는 것이 아니고 계속 퍼블리시를 하고 있었는데, 이번에 갑자기 에러가 걸립니다.

![Screenshot_5](https://user-images.githubusercontent.com/68143858/178868197-1c734b04-f971-47e3-88e2-c67288947324.png)

또한, 테스트를 위해 서버를 키고 테스트할 때도 에러가 걸리네요.

![Screenshot_6](https://user-images.githubusercontent.com/68143858/178868320-5eb29852-ae5a-4322-873c-095e206f907c.png)

왜 이러는지 알 수 있을까요??

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 오류 상황에 대한 보고는 월드 버전, 개발 환경과 함께 이슈 리포트를 통해 문의 부탁 드립니다.
https://github.com/naverz/zepeto-studio-kor/issues
감사합니다.
